### PR TITLE
Add update params check when executing a dapi fallback

### PR DESCRIPTION
**Pull request by alikonuk1**

### CODE REVIEW

File: contracts/DapiFallbackV2.sol [hunk 1]
```diff
@@ -82,6 +82,23 @@ contract DapiFallbackV2 is Ownable, IDapiFallbackV2 {
...
+        require(
+            deviationThresholdInPercentage == 1e6,
+            "Deviation threshold must be 1%"
+        );
+        require(deviationReference == 0, "Deviation reference must be 0");
+        require(
+            heartbeatInterval == 86400,
+            "Heartbeat interval must be one day"
+        );
+
```
Comment (bbenligiray) [inline contracts/DapiFallbackV2.sol:101]: Declare a private constant that is `keccak256(abi.encode(uint256(1e6), int224(0), uint32(1 days)))` and compare `keccak256(args.updateParams)` with that

File: test/DapiFallbackV2.sol.js [hunk 2]
```diff
@@ -124,7 +124,15 @@ describe('DapiFallbackV2', function () {
 
     const duration = 7776000; // 90 days in seconds
     const price = hre.ethers.utils.parseEther('3');
-    const updateParams = generateRandomBytes();
+    const deviationThresholdInPercentage = hre.ethers.utils.parseUnits('1', 6); // 1e6 represents 1%
```
Comment (bbenligiray) [inline test/DapiFallbackV2.sol.js:127]: Test the case where the update parameters are incorrect

**Review by bbenligiray — COMMENTED** [review]
Create the tree validation issue

**Review by acenolaza — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**